modelci/metrics/benchmark/metric.py

```diff
@@ -135,20 +136,24 @@ def run_model(self, server_name: str, device: str):
         # FIXME: if the number of batch is really small, the GPU utilization will get a smaller value.
         # Usually, in order to increase accuracy, we need to increase the testing number of batches, try to make sure
         # the testing program can run over 1 minutes.
-        cadvisor = CAdvisor()
-        SLEEP_TIME = 15
-        time.sleep(SLEEP_TIME)
-        all_information = cadvisor.request_by_name(server_name)
-        model_info = cadvisor.get_model_info(all_information)
-        stats = model_info[list(model_info.keys())[0]]['stats']
-        val_stats = [x for x in stats[-int(SLEEP_TIME + all_data_latency):] if
-                     x['accelerators'][0]['duty_cycle'] != 0]
-        all_batch_avg_memory_total = sum([i['accelerators'][0]['memory_total'] for i in val_stats]) / len(val_stats)
-        all_batch_avg_memory_used = sum([i['accelerators'][0]['memory_used'] for i in val_stats]) / len(val_stats)
-        all_batch_avg_util = sum([i['accelerators'][0]['duty_cycle'] for i in val_stats]) / len(val_stats)
-        memory_avg_usage_per = all_batch_avg_memory_used / all_batch_avg_memory_total
-
+        all_batch_avg_memory_total = 0
+        all_batch_avg_memory_used = 0
+        all_batch_avg_util = 0
+        memory_avg_usage_per = 0
         if cuda:
```

Comment:
How about CPU device?

Reply:
Actually, we do not have a memory monitor for cpu before. This PR is mostly about smoothing the process of profiling and connecting it to our new persistence module. Maybe we should add a TODO here.

Reply:
Ok, add a TODO.